lib/matplotlib/text.py

```
@@ -739,6 +746,13 @@ def draw(self, renderer):
             gc.set_url(self._url)
             self._set_gc_clip(gc)
 
+            text_aa = self.get_antialiased()
+            resolved_text_aa = (
+                text_aa if text_aa is not None
+                else mpl.rcParams['text.antialiased'])
+            if hasattr(gc, "set_text_antialiased"):
```

[major] The `hasattr(gc, "set_text_antialiased")` guard is still here. Now that `GraphicsContextBase` defines the method, this is duck typing in the core logic and will quietly mask backends that forgot to override it. Please remove the guard and invoke `gc.set_text_antialiased(resolved_text_aa)` unconditionally so missing implementations fail fast.